source/code/data_types.md

```diff
@@ -33,117 +33,73 @@ Verses in single chapter books should always be referenced using 1 as the chapte
+<hr />
 
-```html
-<hr data-parsing="Gen.1" />
-<!-- OR -->
-<span data-parsing="Gen.1"></span>
-```
+## Scripture References
 
-### Do Not Parse
+Scripture references use the full b.c.v notation, and just as with the Scripture context attribute, a reference to more than one verse must be b.c.v-b.c.v.
 
-To indicate that a specific portion of text should not be parsed as Scripture, use `data-parsing` along with an empty value. Remember to use with a `<span>` element, rather than `<hr>`.
+_**All Scripture references in each volume must be tagged, non-orphaned and orphaned references alike.**_
 
 ```html
-<span data-parsing=""></span>123:4<span data-parsing="Gen"></span>
+<a data-ref="Rom.3.23">Romans 3:23</a>
+...
+<a data-ref="Acts.1.1-Acts.2.47">Acts 1—2</a>
+...
+<a data-ref="Jer.33.3">33:3</a>
+...
+<a data-ref="Gen.1.1-Gen.1.2">verses 1 and 2</a>
 ```
 
-<aside class="warning">Important: After using an empty parsing attribute in this way, always restore context with a normal parsing context tag, as shown above.</aside>
+<aside class="notice">There is no need to add Scripture reference tags to Old Testament Apocrypha, New Testament Apocrypha, or Pseudepigrapha references.</aside>
 
-### Date Context
+## Workbook Questions and Answers
 
-Date context for books that are date-aligned, like some devotionals, is identified by `data-datetime`, and its value is an HTML datetime value.
+Workbook enhancements are a Content Platform (Wordsearch web app) feature, used for marking question text and answer locations wherever it is desirable to give the user a text box in which to record their answer to a question or exercise. See [Dev Process > Workbook Enhancements](/process/special_types.html#Workbook-Enhancements-In-Content-Platform) for more info on this feature.
 
-For most books, month and day values are sufficient:
+The element containing the question text should receive the `data-wb-question` attribute where the value is an identifier unique to that question. (Each `data-wb-question` value must be unique within the EPUB)
 
 ```html
-<!-- Just the month and day -->
-<hr data-datetime="mm-dd" />
-```
+<p class="ind1" data-wb-question="chap14-1">1.  How does Luke describe Athens and its people (17:16, 21)?</p>
+<p class="ind1" data-wb-question="chap14-2">2.  What did the cultured men think of Paul, and why (17:18, 32)?</p>
 
-If the devotional has morning and evening entries, AM and PM can be specified in the date attribute:
+<!-- OR -->
 
-```html
-<!-- Date with AM or PM -->
-<hr data-datetime="mm-ddT00:00" /> <!-- for AM -->
-<hr data-datetime="mm-ddT12:00" /> <!-- for PM -->
+<p>Abraham believed two things about God that convinced him God could keep His promise (4:17)</p>
+<ol class="lower-alpha">
+  <li data-wb-question="chap06-9">In what ways does God give "life to the dead" (4:17)? </li>
+  <li data-wb-question="chap06-10"> How does God call "things that are not as though they were" (4:17)?</li>
```

Comment:
I know this is really tiny but I noticed there is a space before the `</li>` on line 70 and a space after the `<li data-wb-question="chap06-10">` on line 71.

Reply:
Thanks, got those fixed.